build.gradle

```diff
@@ -15,14 +15,13 @@
  */
 
 import org.gradle.build.Install
-import org.gradle.build.DefaultJavaInstallation
 import org.gradle.cleanup.EmptyDirectoryCheck
-
-import org.gradle.jvm.toolchain.internal.JavaInstallationProbe
+import org.gradle.modules.PatchExternalModules
 
 plugins {
     id 'java-base'
     id 'build-types'
+    id 'gradleCompile'
```

Comment:
:o: Let's use same case for all `buildSrc` plugin IDs: `gradle-compile`

Reply:
Done.